.binder/requirements.txt

```diff
@@ -2,3 +2,8 @@ itk-ultrasound>=0.3.2
 itkwidgets
 matplotlib
 requests
+zarr
+xarray
+pandas
+seaborn
+sklearn
```

Comment:
I left out numpy, counting that https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/159 will get merged eventually, and most people already have numpy anyway.